Update _index.md (#9435)
cloudflare/cloudflare-docs: content/fundamentals/notifications/_index.md

* Free plans can set up email-based Notifications.
* Business and higher plans can also [access PagerDuty](/fundamentals/notifications/create-notifications/create-pagerduty/).
* Professional and higher plans can also [use webhooks](/fundamentals/notifications/create-notifications/configure-webhooks/).
  

> The notification service only works on the **proxied domains**, so we can gather the information necessary to decide if we need to trigger a notification or not.

Reviewer: ```suggestion
The notification service only works on the [proxied](/dns/manage-dns-records/reference/proxied-dns-records/) domains because Cloudflare needs enough information necessary to decide if we need to trigger a notification or not.
```
